### PR TITLE
add aurora

### LINKED ISSUE #5168: [Go] Add aurora
Hi @aurora-go,

Would you mind to add (or consent that we add) [aurora](https://github.com/aurora-go/aurora) in `here` ?

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

### PR DESCRIPTION
**Pull request by waghanza**
Hi @aurora-go,

This `PR` adds `aurora` in here.

However, not sure about how to get `id` URL param in `/user/{id}` 

Also, may I notify you on each update (of `aurora`) in here ?

-------
Closes #5168

### CODE REVIEW

**Comment by aurora-go** [comment]
Two steps, args: = request [req. Args] (map [string] interface {}), then args ["Id"]. I wonder if it is necessary to write an English document and update it in the near future

**Comment by waghanza** [comment]
Not sure I understand which documentation you are talking about

File: go/aurora/main.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+func main() {
+	//获取 aurora 路由实例
+	a := aurora.New()
+
+	a.GET("/", func(request aurora.Request) interface{} {
+		return ""
+	})
+	a.GET("/user/{id}", func(request aurora.Request) interface{} {
+		return request[req.Args]
```
Comment (waghanza) [inline go/aurora/main.go:16]:
```suggestion
		args := request[req.Args].(map[string]interface{})
		return args["id"]
```

**Comment by aurora-go** [comment]
It's the API instructions. Yesterday, I checked the document and missed the part about URL parameter acquisition. Now it seems that it has been completed and merged. It's embarrassing. I have to improve the API and use instructions.

**Comment by waghanza** [comment]
Feel free to open a new to enhance the code

**Pull request merged**